teleop_twist_joy_node:
  ros__parameters:
    axis_linear:
      x: 5
    scale_linear:
      x: 1.0

    axis_angular:
      yaw: 0
    scale_angular:
      yaw: 1.0

    axis_linear_reverse:
      x: 2
    scale_linear_reverse:
      x: 1.0

    publish_stamped_twist: False

    require_enable_button: False
